docs/containers/mysql.md

```diff
@@ -0,0 +1,44 @@
+# Mysql
+
+This is information about the mysql service :
+
+- password roor : `nextcloud`
+
+- user : `nextcloud`
+- password : `nextcloud`
+
+- database name : `nextcloud`
+
+## How to connect on the mysql service ?
+
+First, found the name of your container :
```

Comment:
This can be much shorter with 

`docker compose exec database-mysql mysql -uroot -pnextcloud`

Reply:
> This can be much shorter with
> 
> `docker compose exec database-mysql mysql -uroot -pnextcloud`

I tried and... What is this black magic ?!!! :eyes: 

It's work ! But I recommended using a no root account to respect the best practice.

What do you think ?

Reply:
I created a commit for this change : https://github.com/juliushaertl/nextcloud-docker-dev/pull/87/commits/66251329279419ebaf5d69147240c1e55dec24d5

What do you think ?